src/main/java/org/opensearch/ad/ml/ThresholdingResult.java

```diff
@@ -21,9 +24,20 @@
     private final double grade;
     private final double confidence;
     private final double rcfScore;
+    private long totalUpdates;
+    private boolean startOfAnomaly;
+    private boolean inHighScoreRegion;
+    private int relativeIndex;
+    private double[] currentTimeAttribution;
+    private double[] oldValues;
+    private double[][] expectedValuesList;
```

Comment:
How about add some java doc for some fields not easy to understand like `inHighScoreRegion`, `relativeIndex`, `oldValues`, `expectedValuesList` ?

Reply:
Never mind, find the java doc in AnomalyResult.java.